controllers/news.controllers.js

```diff
@@ -1,7 +1,16 @@
-const { selectTopics } = require("../models/news.models")
+const { selectTopics, selectArticles, selectArticle } = require("../models/news.models")
 
 exports.getTopics = (req, res, next) => selectTopics().then((topics) => {
-        return res.status(200).send({topics})
+        res.status(200).send({ topics })
     }).catch((err) => {
         next(err)
```

Comment:
for a little syntatic-sugar , if you write the catch block as `.catch(next);` express will read this the same way and pass the err along to the app - a nice little shorthand